lib/tictactoe.rb

```diff
@@ -5,6 +5,7 @@ def initialize(io:)
     @board = Array.new(9, ' ')
     @current_token = 'X'
     @io = io
+    @move_valid = false
```

Comment:
It's a bit weird to have to reset this attribute to false each time you switch a turn. Is it possible to just have a method that evaluates if a move is valid or not and use that to check?

Reply:
That's a good point. I'll find a better way to do it.